### ISSUE TITLE
insufficient color contrast on documentation pages

### ISSUE THREAD

**Issue by mwarner** [axios/axios-docs#294]
In light mode, many text elements have 2.95:1 contrast ratio. Dark mode is better, but still short of the mark at 3.94:1. The minimum is 4.5:1. This negatively impacts users with even moderate visual impairment.

Please see https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html for more information.

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/8317a5a6-4238-405a-bee2-159a228f95af" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/92b86ded-fe1d-41b3-9b69-39758c8c8465" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/445c545b-915b-4246-840b-1677ccd278ca" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/fcf594e0-2cda-4cf4-bf13-02b3946620b6" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/a58f33b3-1498-4738-885b-c2361071e08a" />

**Comment by jasonsaayman** [comment]
Thanks for this issue. This repo will be deprecated and archived, as we will keep the docs in the main repo going forward.
